src/ServiceControl.Audit/Auditing/MessagesView/GetMessages2Controller.cs

```diff
@@ -0,0 +1,57 @@
+namespace ServiceControl.Audit.Auditing.MessagesView;
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Infrastructure;
+using Infrastructure.WebApi;
+using Microsoft.AspNetCore.Mvc;
+using Persistence;
+
+[ApiController]
+[Route("api")]
+public class GetMessages2Controller(IAuditDataStore dataStore) : ControllerBase
```

Comment:
Wouldn't it be possible to use `ApiVersionAttribute` by using the Microsoft.AspNetCore.Mvc.Versioning package?

This is just pseudo typing but it would then become something like:

```suggestion
[ApiController]
[Route("api")]
[ApiVersion("2.0")]
public class GetMessages2Controller(IAuditDataStore dataStore) : ControllerBase
```

This also allows for a more clean usage of the deprecated API later on.

Reply:
Not in this PR

Reply:
@ramonsmits I am not too concerned with api versioning given that we control the clients and this is not a publicly documented api so we can evolve it anyway we want as long our clients do not break. 
So if you feel strongly about api versioning we can have that discussion separately from this work. 
But otherwise can you unblock the PR so we can proceed with the port of SI to SP.